docs/spelling_wordlist.txt

```diff
@@ -161,6 +161,7 @@ BestCandidate
 Bigquery
 bigquery
 BigQueryHook
+BigQueryOperator
```

Comment:
This is not needed, you should mark the place where you use is as code then the spelling check will know to treat it as such.

Reply:
Removed, as it's removed from comment.